rules/ensure-external-secrets-storage-is-in-use/test/failed/expected.json

```diff
@@ -0,0 +1,33 @@
+[{
+    "alertMessage": "Deployment: depName is not using external secret storage",
+    "failedPaths": ["spec.volumes[0].csi.volumeAttributes.secretProviderClass"],
```

Comment:
this path doesn't exist in the input for this test,
did you mean for it to be a fixPath? if so, please move it there and add a value

Reply:
done